includes/template-tags.php

```diff
@@ -608,6 +610,119 @@ function dokan_store_category_menu( $seller_id, $title = '' ) {
     }
 
 endif;
+if ( ! function_exists( 'dokan_store_term_menu_list' ) ) :
+
+    /**
+     * Store category menu for a store
+     *
+     * @param  int $seller_id
+     *
+     * @since DOKAN_SINCE
+     *
+     * @return void
+     */
+    function dokan_store_term_menu_list( $seller_id, $taxonomy, $query_type ) {
+        ?>
+    <div id="cat-drop-stack" class="store-cat-stack-dokan">
+        <?php
+        $seller_id = empty( $seller_id ) ? get_query_var( 'author' ) : $seller_id;
+        $vendor    = dokan()->vendor->get( $seller_id );
+        $store_url = dokan_get_store_url( $seller_id );
+        if ( $vendor instanceof Vendor ) {
```

Comment:
This method implementation looks so unreadable. Also some inside alignments are not okay. Kindly see if it can be improved.